src/parsec_client/operations/ping.md

```diff
@@ -4,7 +4,13 @@
 
 ## Summary
 
-Determines whether the service is present and responsive on the expected endpoint.
+Determines whether the service is present and responsive on the expected endpoint. Clients should
+follow the following bootstrapping sequence if they want to switch to the highest wire protocol
+version that the service support:
+
+1. Client requests a Ping operation using the wire protocol version `1.0`.
+2. Service responds with the highest wire protocol version supported `x.y`.
+3. Client can now use wire protocol version `x.y` for further requests.
```

Comment:
`Client can now use any wire protocol version below x.y` - trying to cover the case where the client might only support a lower version